File: pweatherstation/WSModel.py
class WSModel:
    def __init__(self, altitude=None, luminosity=None, humidity=None, pressure=None, temperature=None, distance=None):
        self._altitude = altitude
        self._luminosity = luminosity
        self._humidity = humidity
        self._pressure = pressure
        self._temperature = temperature
        self._distance = distance

    @property
    def altitude(self) -> float:
        """
        :return: Высота точки измерения относительно места установки метеостанции, м
        """
        return self._altitude

    @altitude.setter
    def altitude(self, value: float):
        """
        :param value: Высота точки измерения относительно места установки метеостанции, м
        """
        self._altitude = value

    def is_altitude_empty(self) -> bool:
        """
        :return: Признак наличия высоты точки измерения относительно места установки метеостанции
        """
        return True if not self._altitude else False

    @property
    def luminosity(self) -> float:
        """
        :return: Уровень освещенности, люмен
        """
        return self._luminosity

    @luminosity.setter
    def luminosity(self, value: float):
        """
        :param value: Уровень освещенности, люмен
        """
        self._luminosity = value

    def is_luminosity_empty(self) -> bool:
        """
        :return: Признак наличия уровня освещенности
        """
        return True if not self._luminosity else False

    @property
    def humidity(self) -> float:
        """
        :return: Относительная влажность, %
        """
        return self._humidity

    @humidity.setter
    def humidity(self, value: float):
        """
        :param value: Относительная влажность, %
        """
        self._humidity = value

    def is_humidity_empty(self):
        """
        :return: Признак наличия относительной влажности
        """
        return True if not self._humidity else False

    @property
    def pressure(self) -> float:
        """
        :return: Давление, мм.рт.ст
        """
        return self._pressure

    @pressure.setter
    def pressure(self, value: float):
        """
        :param value: Давление, мм.рт.ст
        """
        self._pressure = value

    def is_pressure_empty(self)-> bool:
        """
        :return: Признак наличия давления
        """
        return True if not self._pressure else False

    @property
    def temperature(self) -> float:
        """
        :return: Температура, *С
        """
        return self._temperature

    @temperature.setter
    def temperature(self, value: float):
        """
        :param value: Температура, *С
        """
        self._temperature = value

    def is_temperature_empty(self)-> bool:
        """
        :return: Признак наличия температуры
        """
        return True if not self._temperature else False

    @property
    def distance(self):
        """
        :return: Расстрояние до ближайшего сотрудника, см
        """
        return self._distance

    @distance.setter
    def distance(self, value: float):
        """
        :param value: Расстрояние до ближайшего сотрудника, см
        """
        self._distance = value

    def is_distance_empty(self) -> bool:
        """
        :return: Признак наличия высоты точки измерения относительно места установки метеостанции
        """
        return True if not self._altitude else False

    def is_empty(self) -> bool:
        """
        :return: Признак инициализации модели
        """
        return self.is_altitude_empty() and self.is_distance_empty() and self.is_luminosity_empty() and \
               self.is_pressure_empty() and self.is_humidity_empty() and self.is_temperature_empty()

    def to_string(self):
        return("Altitude: {0}, Distance: {1}, Luminosity: {2}, Pressure: {3}, Humidity: {4}, Temperature: {5}".format(
            self.altitude, self.distance, self.luminosity, self.pressure, self.humidity, self.temperature))

    def clear(self):
        self._altitude = None
        self._luminosity = None
        self._temperature = None
        self._humidity = None
        self._pressure = None
        self._distance = None

File: jweatherstation/src/jweatherstation/WSModel.java
package jweatherstation;

public class WSModel {
    private float altitude;
    private float humidity;
    private float luminosity;
    private float pressure;
    private float temperature;
    private float distance;

    public WSModel() {}

    /**
     * @param altitude Высота точки измерения относительно места установки метеостанции, м
     * @param humidity Относительная влажность, %
     * @param luminosity Уровень освещенности, люмен
     * @param pressure Давление, мм.рт.ст
     * @param temperature Температура, *С
     * @param distance Расстояние до пользателя, см
     */
    public WSModel(float altitude, float humidity, float luminosity, float pressure, float temperature, float distance) {
        this.altitude = altitude;
        this.humidity = humidity;
        this.luminosity = luminosity;
        this.pressure = pressure;
        this.temperature = temperature;
        this.distance = distance;
    }

    /**
     * @return Высота точки измерения относительно места установки метеостанции, м
     */
    public float getAltitude() {
        return this.altitude;
    }

    /**
     * @return Относительная влажность, %
     */
    public float getHumidity() {
        return this.humidity;
    }

    /**
     * @return Уровень освещенности, люмен
     */
    public float getLuminosity() {
        return this.luminosity;
    }

    /**
     * @return Давление, мм.рт.ст
     */
    public float getPressure() {
        return this.pressure;
    }

    /**
     * @return Температура, *С
     */
    public float getTemperature() {
        return temperature;
    }

    /**
     * @param altitude Высота точки измерения относительно места установки метеостанции, м
     */
    public void setAltitude(float altitude) {
        this.altitude = altitude;
    }

    /**
     * @param humidity Относительная влажность, %
     */
    public void setHumidity(float humidity) {
        this.humidity = humidity;
    }

    /**
     * @param luminosity Уровень освещенности, люмен
     */
    public void setLuminosity(float luminosity) {
        this.luminosity = luminosity;
    }

    /**
     * @param pressure Давление, мм.рт.ст
     */
    public void setPressure(float pressure) {
        this.pressure = pressure;
    }

    /**
     * @param temperature Температура, *С
     */
    public void setTemperature(float temperature) {
        this.temperature = temperature;
    }

    /**
     * @return Расстояние до пользователя, см
     */
    public float getDistance() {
        return distance;
    }

    /**
     * @param distance Расстояние до пользователя, см
     */
    public void setDistance(float distance) {
        this.distance = distance;
    }

    @Override
    public String toString() {
        return "WSModel{" +
                "altitude=" + altitude +
                ", humidity=" + humidity +
                ", luminosity=" + luminosity +
                ", pressure=" + pressure +
                ", temperature=" + temperature +
                ", distance=" + distance +
                '}';
    }
}